### DISCUSSION TITLE
Website Design Upgrade

### DISCUSSION THREAD

**Discussion by naher94** [jekyll/jekyll#8667] [Ideas]
Would this group be interested in a design update of the https://jekyllrb.com site as well as a brand update? Allow for better A11Y, navigation etc.

Would love to discuss with this group. Big fan of Jekyll! If you are curious of my work: https://rehanbutt.com

**Comment by JackieGable** [comment]
I manage several large websites, all of them written in static php. Only a few have actual blogs, most are just portfolio websites and a few e-commerce. Coming from Wordpress, I hated the constant security concerns associated with databases and poorly written plug-ins, etc.

Recently, I have been trying out various static site generators. Coming from a php background, I have tried Jigsaw and Katana but now I am trying out Jekyll. I must say, I really like the simplicity of Jekyll. 

However, since my dev environment is Windows based, I hate that I am forced to use Ruby in order to run Jekyll. However, the simplicity of Jekyll keeps bringing me back to it.

That said, I am concerned about the future of Jekyll, as it doesn’t appear that anyone is interested in advancing it further. Should I continue with Jekyll or look at other SSGs that are actively maintained?

**Reply by edheltzel** [reply]
Docker is a great option for this
